README.md

```diff
@@ -38,6 +38,11 @@ services like message brokers that exists _somewhere_ in a simple way:
     application configuration primitives in Kubernetes: Services, Secrets, and
     ConfigMaps.
 
+Video links:
```

Comment:
Can we either move the youtube link above down here, or move these up - it seems like they should be next to each other.